dapp/src/components/buySell/ApproveButtonLogic.js

```diff
@@ -112,6 +120,21 @@ const ApproveButtonLogic = forwardRef(
       },
     }))
 
+    const contractName = (contract) => {
+      if (contract === 'flipper') return 'the Flipper'
```

Comment:
Can you wrap the strings in `fbt` so they are translatable? See example here: https://github.com/OriginProtocol/origin-dollar/blob/7074ecfac35b5f362c75afc35fffcce8e15aa63b/dapp/src/components/buySell/SwapHomepage.js#L285-L287